iot-hub/Samples/service/JobsSample/Program.cs

```diff
@@ -3,27 +3,28 @@
 
 using System;
 using System.Threading.Tasks;
+using CommandLine;
+using JobsSample;
 
 namespace Microsoft.Azure.Devices.Samples

```

Comment:
Change this namespace to `JobSample` and you won't need the `using` statement on 7.